tests/Console/ArtisanTest.php

```diff
@@ -96,6 +96,9 @@ public function testArtisanOrchidLink()
 
         $this->artisan('orchid:link')
             ->expectsOutput("The [$resources] directory already exists.");
+
+        $this->artisan('orchid:link --force')
```

Comment:
Let's create a separate test that created a directory and first received an error, and fixing it with --force